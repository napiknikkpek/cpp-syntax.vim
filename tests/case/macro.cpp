#pragma once
#include <string>
# define MY_MACRO_1(x) x+1
#define MY_MACRO(name)                             \
  struct name {                                    \
    const char* operator() const { return #name; } \
  };
auto x = MY_MACRO_1(3);
MY_MACRO(Name)
const int N = 200000;
NONE
