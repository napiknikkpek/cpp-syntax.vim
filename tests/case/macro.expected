ppppppp.pppp
pppppppp.ssssssss
p.pppppp.pppppppppppxp.xpc
ppppppp.pppppppppxxxxp.............................p
..aaaaaa.tttt.p....................................p
....aaaaa.zzzzp.aaaaaaaapp.aaaaa.p.aaaaaa.pppppp.p.p
..pp
aaaa.x...pppppppppp.c..
pppppppp.xxxx.
aaaaa.zzz.x...cccccc.
pppp
